lib/node_modules/@stdlib/stats/base/dists/uniform/mgf/benchmark/benchmark.js

```diff
@@ -30,18 +31,26 @@ var mgf = require( './../lib' );
 // MAIN //
 
 bench( pkg, function benchmark( b ) {
+	var len;
 	var min;
 	var max;
 	var t;
 	var y;
 	var i;
 
+	len = 100;
+	t = new Float64Array( len );
+	min = new Float64Array( len );
+	max = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		t[ i ] = ( randu() * 20.0 ) - 10.0;
```

Comment:
Would you be able to update the logic to generate values to our latest conventions, as exemplified in e.g. PR https://github.com/stdlib-js/stdlib/pull/4929?